src/components/MAPI/permissions/PermissionsOverview/__tests__/PermissionsOverview.tsx

```diff
@@ -1052,4 +1052,260 @@ describe('PermissionsOverview on GCP', () => {
     expect(await screen.findByText('another category')).toBeInTheDocument();
     expect(screen.queryByText('releases category')).not.toBeInTheDocument();
   });
+
+  it('does not display permission use cases for provider credentials if the provider does not support them', async () => {
+    const mockUseCases: IPermissionsUseCase[] = [
+      {
+        name: 'Some use case with provider credentials',
+        category: 'some category',
+        scope: {
+          namespaces: ['*'],
+        },
+        permissions: [
+          {
+            resources: ['secrets'],
+            apiGroups: [''],
+            verbs: ['*'],
+          },
+        ],
+      },
+      {
+        name: 'Some use case with secrets',
+        category: 'some category',
+        scope: {
+          namespaces: ['*'],
+        },
+        permissions: [
+          {
+            resources: ['secrets'],
+            apiGroups: [''],
+            verbs: ['*'],
+          },
+        ],
+      },
+      {
+        name: 'Some use case mentioning provider credentials',
+        category: 'some category',
+        scope: {
+          namespaces: ['*'],
+        },
+        permissions: [
+          {
+            resources: ['someresource'],
+            apiGroups: ['giantswarm.io'],
+            verbs: ['*'],
+          },
+        ],
+      },
```

Comment:
@kuosandys  Could you please explain the third use case - `Some use case mentioning provider credentials`. What does this use case mean for?

Reply:
Got it, I missed the part about the name check. Thanks for clarification.